corehq/apps/sms/tests/test_backends.py

```diff
@@ -223,13 +223,6 @@ def setUpClass(cls):
         )
         cls.airtel_tcl_backend.save()
 
-        cls.trumpia_backend = TrumpiaBackend(
-            name='TRUMPIA',
-            is_global=True,
-            hq_api_id=TrumpiaBackend.get_api_id()
-        )
-        cls.trumpia_backend.save()
```

Comment:
This should not be removed. The same applies to all code mentioning "trumpia" in this file.

Reply:
SMH, huh what an oversight. Fixed.